appsdk/sdk.go

```diff
@@ -68,36 +72,48 @@ func (sdk *AppFunctionsSDK) SetPipeline(transforms ...func(edgexcontext *appcont
 	return nil
 }
 
-// FilterByDeviceID ...
+// FilterByDeviceID - Specify the devices of interest to filter for data coming from certain sensors.
```

Comment:
These helper functions are one of the refactoring items Trevor had pointed out. Devs can accomplish same thing without them like this:
```    
       edgexSdk.SetPipeline(
		transforms.Filter{FilterValues: deviceIDs}.FilterByDeviceID,
		transforms.Conversion{}.TransformToXML,
		printXMLToConsole,
	)
```
If we keep them, then we should be explicit that they are helper functions, not actually calling the transform function. i.e. it is confusing when look at them from API perspective.  Maybe prefix them with Get, i.e. GetFilterByDeviceID would make it clearer?

Reply:
I think the comments remain the same though?

Reply:
To me the comments make me think the actual filtering is being called. It should start with GetFilterByDeviceID returns the transform FilterByDeviceID function configured to... blah, blah.